string eef_name
float64 x
float64 y
float64 z
bool force_orien
---
bool success
